include/irr/asset/CTexturePacker.h

```diff
@@ -80,7 +80,7 @@ class ITexturePacker
         }
         uint32_t x() const { return addr&X_MASK; }
         uint32_t y() const { return (addr>>(ADDR_LAYER_SHIFT>>1))&X_MASK; }
-        uint32_t layer() const { return (addr>>ADDR_LAYER_SHIFT)&0xffffu; }
+        uint32_t layer() const { return (addr&0xffffu)>>ADDR_LAYER_SHIFT; }
```

Comment:
again, `STexOffset` is not your page pointer (doesn't hold the mip-tail pointer in high bits).

Reply:
why are you telling me what my code is and what isnt XD

Reply:
https://github.com/buildaworldnet/IrrlichtBAW/pull/424#discussion_r392970622